src/scss/components/_targid.scss

```diff
@@ -235,6 +235,11 @@ div.filmstrip {
     overflow: hidden;
     transition: width $mode-animation-time ease;
 
+    // add border between viewWrappers when chooser is hidden
+    &.single-view + .viewWrapper {
+      @extend .border-start;
```

Comment:
Is `.border-start` a bootstrap class? If yes, I think it is worth to mention it in a comment to avoid confusions.